subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/AbstractCrossVersionPerformanceTestRunner.groovy

```diff
@@ -138,92 +138,90 @@ class AbstractCrossVersionPerformanceTestRunner extends PerformanceTestSpec {
         version.replace('+', '')
     }
 
+    static Object resolveVersion(String version, ReleasedVersionDistributions releases) {
+        switch (version) {
+            case GradleVersion.current().getBaseVersion().version:
+                // current version is run by default, skip adding it to baseline
```

Comment:
I don't know either. I assume this is a historical usage that we don't use any more. Let's remove this code to see if anything bad happens. This can also simplify the List/Object return value below.